### DISCUSSION TITLE
Water Mist for 10 Sec

### DISCUSSION THREAD

**Discussion by Incredible-IT** [theyosh/TerrariumPI#903] [Q&A] [answered]
Hello,
Fabulous job on this. So far so good. I will definitely be contributing a little sumtin-sumtin to the donation plate.

I have a 3 enclosure case I made from 12mm PVC. Each enclosure is 2' x 2' x 4'. Two enclosures are horizontal and one is vertical. In the two horizontal enclosures I already have very happy Ball Pythons. In the 3rd I am creating a Tropical Vivarium. Natural substrate, real plants, a water fall, and a bunch of small Anole lizards and geckos. Pictures to follow...

Now the actual question. I have a Rain/Mister setup. Due to the amount of pressure I had to apply to get a good rain/mist I can only run it for seconds. **How do I trigger the relay to run for 10-15 seconds?** So far I have only figured out how to do it for a min. That is WAY too much water. I imagine I will need to do a JSON file, but I have no clue how to do that.

Thanks in advance.

**Comment by theyosh** [comment, marked as answer]
In the area setup you can set the 'Power on time' value to 10 for 10 sec on power.
![image](https://github.com/theyosh/TerrariumPI/assets/8080177/ce700a34-f05a-452d-a7f0-b16cf97c1c2f)

Also, you can play with: https://theyosh.github.io/TerrariumPI/scenarios/#timer-wizard

![image](https://github.com/theyosh/TerrariumPI/assets/8080177/b62f414d-c692-4933-82f3-e79960c31a12)

This will trigger your mister 6 times a day for 10 seconds each time.

I hope this will give some extra info to setup your mister.

**Comment by Incredible-IT** [comment]
Thanks @theyosh. I don't know why I didn't see the "Power on time" before. The Timer Wizard will come in handy for another feature as well.

![image](https://github.com/theyosh/TerrariumPI/assets/26004634/2274bd77-1a7d-43f3-a785-4286563c8e12)

**Reply by theyosh** [reply]
You should also set 50 second settle time. Else the mister will run 6 times in the 1 minute period.
